ui/src/app/settings/components/repos-list/repos-list.tsx

```diff
@@ -125,7 +125,7 @@ interface NewGoogleCloudSourceRepoCredsParams {
 
 export enum ConnectionMethod {
     SSH = 'via SSH',
-    HTTPS = 'via HTTPS',
+    HTTPS = 'via HTTP/HTTPS',
```

Comment:
<img width="350" alt="Screenshot 2025-03-05 at 9 31 00 AM" src="https://github.com/user-attachments/assets/3c0bceba-0cd2-4a70-8d66-fb3be5aeb65a" />

This looks ok...